docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
 
-If you click on that link in the browser you should see:
+On line five, we’re importing the header component from  `../components/Header`, and on line 21, we’re including it by using a self-closing JSX tag, `<Header />`. This is why we see the header component on the homepage, even though at first perhaps it seemed like it was missing from our homepage file (`/src/pages/index.js`).
 
-![Gatsby.js development 404 page](dev-404.png)
+Below, I’ve opened up our [homepage](http://localhost:8000/) and [page 2](http://localhost:8000/page-2/) in separate browser windows, and the header component in my code editor. If we modify the default header text (“Gatsby”) and save the file, we’ll see the text change immediately reflected on both pages. 
 
-What you're seeing is the Gatsby.js development 404 page. Let's do what it says
-and create a React.js page component at `src/pages/page-2.js`.
+![](https://d2mxuefqeaa7sj.cloudfront.net/s_E618395E4841D470F48D0AACF52A3EBAEA74345E0FB4AC285CBC937727755E20_1522600157911_gatsby-header2.gif)
 
-Make the second page component look something like:
 
-```jsx
-import React from "react";
-import Link from "gatsby-link";
+That’s the header. Where’s the page content? On line 28, you’ll see `{children()}`. What?
 
-export default () => (
-  <div>
-    <p>Hello world from my second Gatsby page</p>
-    <Link to="/">back home</Link>
-  </div>
-);
-```
+## What are React “props”?
 
-Save that and now you should be able to click back and forth between the two
-pages!
+We’ll come back to `{children()}`.  First, let’s talk about React “props”. “Props” are a way of passing data around in React; They are properties that are supplied to React components. React components are reusable pieces of user interface (UI) that we define. To make them reusable — or, dynamic — we need to be able to supply them with different data. Let’s take a look at this using our familiar `<Header />` component.
 
-<video controls="controls" autoplay="true" loop="true">
-  <source type="video/mp4" src="/images/clicking-2.mp4"></source>
-  <p>Your browser does not support the video element.</p>
-</video>
+### ✋ Generate the Header’s title text using props, instead
+1. On line 17 of `/src/layouts/index.js`, add an attribute called “title” to the `<Header />` component, and assign it some arbitrary text.
 
-_Challenge_: Using the instructions above as hints, see if you can create a third page and link to it from the home page.
 
-## Interactive page
+    <Header
+      title="Meet React Props!"
+    />
 
-One nice thing about using Gatsby for building websites vs. other tools is that itʼs easier to add interactivity to your pages. React.js was designed for
-Facebook.com and is used on many other world-class web applications.
 
-Let's see how to add interactive elements to our pages. Let's start with a counter.
+2. On line 4 of `/src/components/Header/index.js`, type `props` between the parentheses.
```

Comment:
Ok, after doing this, the site broke (error: siteTitle not defined in src/components/header.js). Also, all these things were on different lines in my starter.

I couldn't go beyond this point because wasn't sure how to fix it!

Reply:
I'll have to rethink using line numbers. I don't think that's maintainable.

Reply:
May need to revisit the idea of having a dedicated repo we can link to different states of the tutorial.

Reply:
Yeah that is starting to sound more necessary